openedx/core/lib/logsettings.py

```diff
@@ -126,6 +128,10 @@ def log_python_warnings():
     warnings.filterwarnings('ignore', 'Setting _field_data is deprecated')
     warnings.filterwarnings('ignore', 'Setting _field_data via the constructor is deprecated')
     warnings.filterwarnings('ignore', '.*unclosed.*', category=ResourceWarning)
+    warnings.filterwarnings('ignore', '.*You can remove default_app_config.*', RemovedInDjango41Warning)
+    warnings.filterwarnings('ignore', '.*Instead access HTTPResponse.headers directly.*')
+    warnings.filterwarnings('ignore', '.*Using or importing the ABCs from \'collections\' instead of from \'collections.abc\' is.*')
+    warnings.filterwarnings('ignore', '.*Deprecated call to \`pkg_resources.declare_namespace.*')
```

Comment:
Nit: do we need the "catch-all" prefix for this warning?

Reply:
probably not